supporter-product-data/src/main/scala/com/gu/services/DynamoDBService.scala

```diff
@@ -0,0 +1,72 @@
+package com.gu.services
+
+import com.gu.aws.ProfileName
+import com.gu.model.FieldsToExport._
+import com.gu.model.Stage
+import com.gu.model.dynamo.SupporterRatePlanItem
+import software.amazon.awssdk.auth.credentials.{AwsCredentialsProviderChain, EnvironmentVariableCredentialsProvider, InstanceProfileCredentialsProvider, ProfileCredentialsProvider}
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient
+import software.amazon.awssdk.services.dynamodb.model.{AttributeValue, UpdateItemRequest}
+
+import java.time.{LocalDate, ZoneId, ZoneOffset}
+import java.util.concurrent.CompletionException
+import scala.collection.JavaConverters._
+import scala.compat.java8.FutureConverters._
+import scala.concurrent.{ExecutionContext, Future}
+
+
+class DynamoDBService(client: DynamoDbAsyncClient, tableName: String) {
+
+  def writeItem(item: SupporterRatePlanItem)(implicit executionContext: ExecutionContext) = {
```

Comment:
for me loads of the code gets red underlined with a warning, does that happen for you?  I think it did offer to use the scala linter rules when I first imported so that may be why
![image](https://user-images.githubusercontent.com/7304387/107064883-fa4c5900-67d3-11eb-900d-3b8da1325bdf.png)